lib/api/apiUtils/object/removeAWSChunked.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Checks for V4 streaming value 'aws-chunked' and removes it if present in
+ * Content-Encoding to be compatible with AWS behavior. See:
+ * http://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-streaming.html
+ * @param {string} sourceHeader - Content-Encoding header from request headers
+ * @return {string} new value with 'aws-chunked,' substrings removed
```

Comment:
This seems to only match the first substring 'aws-chunked,'. 

It sounds like the intention here is to match all occurrences of the substring because in the function documentation of the return value 'substrings' is plural. Maybe it is actually only necessary to match the first instance of 'aws-chunked,' though. Otherwise, to match all substrings perhaps you could use `sourceHeader.replace(/aws-chunked,/g, '');`